codex-rs/apply-patch/src/main.rs

```diff
@@ -0,0 +1,40 @@
+use std::io::Write;
+use std::process::ExitCode;
+
+fn main() -> ExitCode {
+    // Expect exactly one argument: the full apply_patch payload.
```

Comment:
This is not quite right: if zero arguments are specified, it is supposed to read from stdin.